BungeeCord-Patches/0042-Fix-potion-race-condition-on-forge.patch

```diff
@@ -0,0 +1,335 @@
+From 12516b0a27dfa89bf7aeec22c16dab2f77d1800f Mon Sep 17 00:00:00 2001
+From: Aaron Hill <aa1ronham@gmail.com>
+Date: Wed, 3 Aug 2016 18:19:25 -0400
+Subject: [PATCH] Fix potion race condition on forge
```

Comment:
Please note in the patch text this only occurs in 1.8.9, so we can easily move it to travertine once upstream drops 1.8.9 support.